.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,11 +1,10 @@
 Fixes #
 
-Changes made in this Pull Request:
- - 
+Changes made in this pull request:
+ -
 
 
 PR Checklist
 ------------
- - [ ] CHANGELOG updated (added [news fragment](https://github.com/hawkowl/towncrier#news-fragments) into changelog directory)?
+ - [ ] Updated `CHANGELOG.rst` ([more information](https://github.com/bio-phys/MDBenchmark/blob/master/DEVELOPER.rst#using-towncrier))?
```

Comment:
Maybe say `added changelog fragment in /changelog` and then the towncrier link

Reply:
Actually, your description in the developer guides looks nice. Be a bit more specific in the dev guides and then link to them from here. That would good.